README.md

```diff
@@ -98,7 +149,19 @@ app.use(bodyParser.urlencoded({extended: true}))
 ```
 
 ```text
-Your answer...
+body-parser - helps is used in-order to read HTTP Post data
+            - is a piece of express middleware that reads a forms input and stores it as a javascript object accessible through req.body
+
+var bodyParser = require("body-parser")
+1. requires a module named body-parser to be loaded, for this to work we first have to save in our app(npm install --save body-parser)
+
+app.use(bodyParser.json())
```

Comment:
The `.json` line enables our application to accept incoming JSON. This line does not have anything to do with forms.